lib/templates/structure/comments.php

```diff
@@ -4,62 +4,69 @@
  *
  * This template will return empty if the post which is called is password protected.
  *
- * @package Structure\Comments
+ * @package Beans\Framework\Templates\Structure
+ *
+ * @since 1.0.0
  */
 
 // Stop here if the post is password protected.
 if ( post_password_required() ) {
 	return;
 }
 
-beans_open_markup_e( 'beans_comments', 'div', array( 'id' => 'comments', 'class' => 'tm-comments' . ( current_theme_supports( 'beans-default-styling' ) ? ' uk-panel-box' : null ) ) );
+beans_open_markup_e(
+	'beans_comments',
+	'div',
+	array(
+		'id'    => 'comments',
+		'class' => 'tm-comments' . ( current_theme_supports( 'beans-default-styling' ) ? ' uk-panel-box' : null ),
+	)
+);
 
-	if ( comments_open() || get_comments_number() ) :
+if ( comments_open() || get_comments_number() ) :
 
-		if ( have_comments() ) :
+	if ( have_comments() ) :
 
-			beans_open_markup_e( 'beans_comments_list', 'ol', array( 'class' => 'uk-comment-list' ) );
+		beans_open_markup_e( 'beans_comments_list', 'ol', array( 'class' => 'uk-comment-list' ) );
 
-				wp_list_comments( array(
+			wp_list_comments(
+				array(
 					'avatar_size' => 50,
 					'callback'    => 'beans_comment_callback',
-				) );
-
-			beans_close_markup_e( 'beans_comments_list', 'ol' );
+				)
+			);
 
-		else :
+		beans_close_markup_e( 'beans_comments_list', 'ol' );
+else :
 
-			/**
+		/**
 			 * Fires if no comments exist.
 			 *
 			 * This hook only fires if comments are open.
 			 *
 			 * @since 1.0.0
 			 */
-			do_action( 'beans_no_comment' );
-
-		endif;
+		do_action( 'beans_no_comment' );
+endif;
 
-		/**
+	/**
```

Comment:
The rest of this comment needs to align left too to align with its opening `/**`.